news/4821-include-file-hash

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* Include file path in addition to the content when generating the file hash. (#4821)
```

Comment:
```suggestion
* Include file path in addition to the content when generating the file hash to avoid unwanted caching during linkage analysis. (#4821)
```